x-pack/plugins/aiops/public/components/log_categorization/category_table/category_table.tsx

```diff
@@ -84,11 +89,16 @@ export const CategoryTable: FC<Props> = ({
     [onAddFilter, onClose]
   );
 
+  const showSparkline = useMemo(() => {
+    return categories.some((category) => category.sparkline !== undefined);
+  }, [categories]);
+
   const openInDiscover = (mode: QueryMode, category?: Category) => {
```

Comment:
Can you use the same tooltip text as we have in the ML AIOps page, where `in Discover` is appended to the text? Otherwise it implies the filter action is applied to the ML page.

<img width="166" alt="image" src="https://github.com/elastic/kibana/assets/7405507/1d316269-b222-4696-9d1b-812d3499533e">

Reply:
Updated in https://github.com/elastic/kibana/pull/169400/commits/da8d015b669a35d4eda8e1732fc09fde2c5d3872